### PR TITLE
[12.0][FIX] account_asset_management - Allow editing salvage value

### LINKED ISSUE #844: [12.0] account_asset_management : Salvage value is not editable when asset created by vendor bills
Salvage Value can not edit when user created by vendor bills

user choose asset profile on vendor bills
Selection_001

Odoo auto generates asset and salvage value can not edit. However users want to edit salvage value before confirm asset.
Selection_002

Comented here: https://github.com/OCA/account-financial-tools/pull/756

### PR DESCRIPTION
**Pull request by Saran440**
Fix bug issue #844
Editable salvage value when created by vendor bills

### CODE REVIEW

**Review by luc-demeyer — APPROVED** [review]

**Review by rafaelbn — APPROVED** [review]

**Comment by newtratip** [comment]
Hello @Saran440 
__manifest__.py is conflict, Can you resolve ?

**Comment by Saran440** [comment]
@newtratip Fixed.

**Comment by kittiu** [comment]
Can this one be merged?

**Comment by newtratip** [comment]
Conflict @Saran440

**Comment by ivantodorovich** [comment]
:+1: but the conflicts are still there

**Review by ivantodorovich — APPROVED** [review]

**Comment by luc-demeyer** [comment]
👍 all green now hence I think we should merge.

File: account_asset_management/static/description/index.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
 <html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
 <head>
 <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
-<meta name="generator" content="Docutils 0.15.1: http://docutils.sourceforge.net/" />
+<meta name="generator" content="Docutils 0.14: http://docutils.sourceforge.net/" />
```
Comment (pedrobaeza) [inline account_asset_management/static/description/index.html:6]: @Saran440 please next time update your docutils version for matching the one in OCA and avoid this useless diff.

**Pull request merged**